--- karma.conf.js	2023-12-12 10:18:25.041323604 +0100
+++ karmanew.conf.js	2023-12-17 14:37:37.702912846 +0100
@@ -2,32 +2,33 @@
 // https://karma-runner.github.io/0.13/config/configuration-file.html
 
 module.exports = function (config) {
-  config.set({
-    basePath: '',
-    frameworks: ['jasmine', '@angular/cli'],
-    plugins: [
-      require('karma-jasmine'),
-      require('karma-chrome-launcher'),
-      require('karma-jasmine-html-reporter'),
-      require('karma-coverage-istanbul-reporter'),
-      require('@angular/cli/plugins/karma')
-    ],
-    client:{
-      clearContext: false // leave Jasmine Spec Runner output visible in browser
-    },
-    coverageIstanbulReporter: {
-      reports: [ 'html', 'lcovonly' ],
-      fixWebpackSourcePaths: true
-    },
-    angularCli: {
-      environment: 'dev'
-    },
-    reporters: ['progress', 'kjhtml'],
-    port: 9876,
-    colors: true,
-    logLevel: config.LOG_INFO,
-    autoWatch: true,
-    browsers: ['Chrome'],
-    singleRun: false
-  });
-};
+    config.set({
+      basePath: '',
+      frameworks: ['jasmine', '@angular/cli'],
+      plugins: [
+        require('karma-jasmine'),
+        require('karma-chrome-launcher'),
+        require('karma-jasmine-html-reporter'),
+        require('karma'),
+        require('@angular-devkit/build-angular/plugins/karma')
+      ],
+      client:{
+        clearContext: false // leave Jasmine Spec Runner output visible in browser
+      },
+      coverageIstanbulReporter: {
+        reports: [ 'html', 'lcovonly' ],
+        fixWebpackSourcePaths: true
+      },
+      angularCli: {
+        environment: 'dev'
+      },
+      reporters: ['progress', 'kjhtml'],
+      port: 9876,
+      colors: true,
+      logLevel: config.LOG_INFO,
+      autoWatch: true,
+      browsers: ['Chrome'],
+      singleRun: false
+    });
+  };
+  
\ No newline at end of file
